frontend/test/metabase/scenarios/admin/settings/sso.cy.spec.js

```diff
@@ -0,0 +1,22 @@
+import { restore } from "__support__/e2e/cypress";
+
+describe("scenarios > admin > settings > SSO", () => {
+  beforeEach(() => {
+    restore();
+    cy.signInAsAdmin();
+  });
+
+  it.skip("Google sign-in client ID should save on subsequent tries (metabase#15974)", () => {
+    cy.visit("/admin/settings/authentication/google");
+    cy.findByPlaceholderText("Your Google client ID").type("123");
+    saveSettings();
+
+    cy.findByDisplayValue("123").type("456");
```

Comment:
Since we're likely going to enforce inputting full client id, then can you change 123 and 456 to a syntax like this:
`123456789012-345678abcdefghijklmnopqrstuvwxyz.apps.googleusercontent.com`